notebooks/MosvizNIRISSExample.ipynb

```diff
@@ -82,20 +94,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Run this cell if you wish to download and unzip our example data.\n",
-    "# This requires the \"zipfile\" package.\n",
-    "\n",
-    "from zipfile import ZipFile\n",
-    "from astropy.utils.data import download_file\n",
-    "import pathlib\n",
-    "\n",
-    "example_data = 'https://stsci.box.com/shared/static/9lkf5zha6zkf8ujnairy6krobbh038wt.zip'\n",
+    "example_data = 'https://stsci.box.com/shared/static/k94suj9yv9vdbusmcdrcodhxt9urwevr.zip'\n",
```

Comment:
tl;dr -- Why not MAST download here?

Reply:
It's because the full observation has a ton of objects in the catalog and is way too slow to load. This is a version of the data made by @camipacifici that cuts down the number of objects to something manageable.